Fundsoft/report.md

```diff
@@ -127,10 +127,44 @@ Optional (point 2): relation to design pattern(s).
 
 ## Overall experience
 
+Overall work
+
+Our contribution to the open source library pdm can be summarized as the following:
+
+### Chosen issue and open source interaction
+Chosen issue from pdm which was assigned to Rasmus (rasmus-d): https://github.com/pdm-project/pdm/issues/2628 
+
+This issue intends to tackle a problem with pdm where a user cannot update a single sub-dependency. Before our patch is was only possible for a user to update a single dependency, all dependencies or all dependencies + all subdependencies but not a single sub-dependecy. Thus the issue relates to implementing a feature where a user can update a specific sub-dependency without changing/updating any other dependency or subdependency. 
```

Comment:
Isn't it possible also to update a dependency + its sub-dependencies?

Reply:
Yes it is! Good catch